src/linagora.esn.contact/app/services/live-update/contact-live-update.service.spec.js

```diff
@@ -12,18 +12,16 @@ describe('The ContactLiveUpdate service', function() {
   var bookId = 'A bookId';
 
   beforeEach(function() {
-    module('esn.core');
-    module('esn.websocket');
-    module('esn.api-notification');
-    module('linagora.esn.contact');
+    angular.mock.module('esn.core');
+    angular.mock.module('esn.websocket');
+    angular.mock.module('esn.api-notification');
+    angular.mock.module('linagora.esn.contact');
   });
 
   beforeEach(function() {
     getMock = function() {};
     session = {
-      ready: {
-        then: function() {}
-      },
+      ready: Promise.resolve({}),
```

Comment:
`$q.when({})` should work here. Tbh I think we should not mix `Promise` and `$q` in AngularJS. We should just only use `$q`.